index.bs

```diff
@@ -385,9 +385,16 @@ enum XREnvironmentBlendMode {
   "alpha-blend",
 };
 
+enum XRVisibilityState {
+  "visible",
+  "visible-blurred",
+  "hidden",
+};
+
 [SecureContext, Exposed=Window] interface XRSession : EventTarget {
   // Attributes
   readonly attribute XREnvironmentBlendMode environmentBlendMode;
+  readonly attribute XRVisibilityState visibilityState;
```

Comment:
[SameObject]?

Reply:
I don't believe that's applicable here, because the state changes over time. (Also not sure if `[SameObject]` is applicable to enums at all.)